### PR TITLE
Add Template of Bug Report

### PR DESCRIPTION
**Pull request by crcrpar**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Create a report to help us improve
```
Comment (eqy) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]:
```suggestion
about: Create a report to help us improve apex
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the Bug**
+
+**Minimal Steps/Code to Reproduce Bug**
```
Comment (eqy) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]:
```suggestion
**Minimal Steps/Code to Reproduce the Bug**
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+<!--
+Please list *minimal* steps or code snippet for us to be able to reproduce the bug.
```
Comment (eqy) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]:
```suggestion
Please list the *minimal* steps or provide a code snippet for us to be able to reproduce the bug.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+A  helpful guide on on how to craft a minimal bug report  http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports.
```
Comment (eqy) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]:
```suggestion
A helpful guide on on how to craft a minimal bug report http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+--> 
+
+**Expected Behavior**
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Environment**
+<!-- OS, version of Python, CUDA, PyTorch, you can easily collect these by running The output of `python -m torch.utils.collect_env` -->
```
Comment (eqy) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]:
```suggestion
<!-- OS, version of Python, CUDA, PyTorch; collect these via `python -m torch.utils.collect_env` -->
```

**Review by eqy — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+**Minimal Steps/Code to Reproduce the Bug**
+<!--
+Please list the *minimal* steps or provide a code snippet for us to be able to reproduce the bug.
+
+A helpful guide on on how to craft a minimal bug report http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports.
+--> 
+
+**Expected Behavior**
+<!-- A clear and concise description of what you expected to happen. -->
```
Comment (zhuoxunyi) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]: [zhuohai346@gmail.com](url)

**Pull request merged**